build.gradle.kts

```
@@ -4,9 +4,7 @@ plugins {
     // https://maven.fabricmc.net/fabric-loom/fabric-loom.gradle.plugin/
     id("fabric-loom") version("1.7.3") apply(false)
     // https://projects.neoforged.net/neoforged/neogradle
```

```suggestion
    // https://projects.neoforged.net/neoforged/moddevgradle
```
these URLs are intended to point to the place to find the latest version numbers (like `2.0.26-beta` here)